std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -996,22 +996,24 @@ struct BitmappedBlockWithInternalPointers(
     }
 
     /// Ditto
-    void[] resolveInternalPointer(void* p)
+    Ternary resolveInternalPointer(void* p, ref void[] result)
     {
+        result = null;
```

Comment:
On third thought: we are under no obligation to assign anything to result if we return anything but `Ternary.yes`, so please remove this line entirely!